infection.json.dist

```diff
@@ -0,0 +1,17 @@
+{
+    "source": {
+        "directories": [
+            "src"
+        ],
+        "excludes": [
+            "test/**/*.php",
+            "vendor/**/*.php"
+        ]
+    },
+    "logs": {
+        "text": "infection.log"
```

Comment:
IMO, infection should:

 * print to STDERR 
 * print in github output format

For an example, see https://github.com/Roave/BackwardCompatibilityCheck/blob/313afe03be5ee64f4a9c8d037b1f42b9d04175b2/infection.json.dist#L7-L13

The stryker setup should probably also be done: the API key will be configured later (needs to be done per-repository, so that may be a subsequent detail)